### PR TITLE
[Documentation][Cookbook] Custom translatable model

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | no |
| New docs?    | yes |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
This looks really good, I will give some time for others to review, but I approve. 👍

File: docs/cookbook/custom-translatable-model.rst [hunk 1]
```diff
@@ -0,0 +1,529 @@
...
+    use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+    use Symfony\Component\Form\Extension\Core\Type\TextType;
+    use Symfony\Component\Form\FormBuilderInterface;
+
+    class SupplierType extends AbstractResourceType
+    {
+        /**
+         * @var TranslationLocaleProviderInterface
+         */
+        private $localeProvider;
```
Comment (NeverResponse) [inline docs/cookbook/custom-translatable-model.rst]: why?

File: docs/cookbook/custom-translatable-model.rst [hunk 2]
```diff
@@ -0,0 +1,529 @@
...
+         * @var string
+         */
+        private $name;
+
+        /**
+         * @var string
+         */
+        private $description;
+
+
```
Comment (NeverResponse) [inline docs/cookbook/custom-translatable-model.rst]: doubled new line

File: docs/cookbook/custom-translatable-model.rst (hunk 1 above)
Comment (TheMadeleine) [inline docs/cookbook/custom-translatable-model.rst]: Wasn't sure if it's needed :)

**Comment by pjedrzejewski** [comment]
Nice, thank you Magda! :)

**Pull request merged**